docs/en/quick-answers/troubleshooting/performance-troubleshooting.md

```diff
@@ -0,0 +1,223 @@
+    - The hierarchy on the File Open app can also be configured to defer the loading of the [sub items until it is expanded](https://support.shotgunsoftware.com/hc/en-us/articles/219033088-Your-Work-Files#Deferred%20queries). This is now the default configuration setup, however, if you have older configs you may wish to transition over to using this.
+    - Check that debug logging isn’t enabled. This can cause a lot of additional I/O and therefore slow things down; these apps do contain a lot of debugging output.
+- **Is it slow opening, saving, or creating a new file?**
+    - Check to see if you’ve taken over scene operations or actions hooks, and see if there is any custom behavior around these functions that might slow things down.
+    - When creating or saving a file, Workfiles will ensure that all the required folders for the context are created. Folder creation can be a common point at which performance [issues](#folder-creation-is-slow) can occur.
+
+## Folder Creation is slow
+
+Folder creation has many parts to it, which can contribute to the process being slow when an issue arises.
+
+Folder creation will:
+- Synchronise your local path cache.
+- Read your config’s schema.
+- Generate a list of paths that should be created given a certain context. 
+- Check the paths against path registries stored locally.
+- Attempt to register the new paths both on your SG site and locally, if not already registered.
+- Check to see if the folders actually exist on disk regardless of if they have already been registered, and create the folders if they are not.
+
+In short, folder creation will have potentially significant I/O usage on the disk, as well as need to write to a local database and communicate with the SG site.
+
+### Tackling I/O usage
+
+It may be possible that your storage is slow or inefficient at handling many small read-write operations, so anything that can be done to improve the infrastructure will help speed up the folder creation operations. However, there are steps that can be taken on the Toolkit configuration side to try and reduce the strain as much as possible.
+
+The first thing is to limit the folders that are created to the ones that are important to that context and thus the environment you would be working in. For example, if you're working on a Task on a Shot in Maya, then you would ideally only want it to check and create the folders for your specific Shot and software. 
+
+Basically, create the minimum required folders that allow you to save and publish your work.
+
+##### Create with parent
```

Comment:
A few weird things with headers:
 - "Create with parent" doesn't seem to be a header at all, but maybe that's what h5 looks like
 - "Defer creation" looks like it maybe should be at the same level as "Create with parent", but it looks different
 - the "Example" header under Create with parent doesn't have its own line when it's render in html – probably need a blank line after it in markdown. 

I did not actually read this article because I *think* I read it back in its Google doc incarnation. If you want to me to actually go through it, I'm happy to.

Reply:
Ah yes good spot, the create with parent had too many `#` it should have been the same as defer creation. I've added a new line after example.